src/ctap/storage.rs

```diff
@@ -397,9 +395,13 @@ impl PersistentStore {
     }
 
     pub fn pin_retries(&self) -> u8 {
-        self.store
-            .find_one(&Key::PinRetries)
-            .map_or(MAX_PIN_RETRIES, |(_, entry)| entry.data[0])
+        match self.store.find_one(&Key::PinRetries) {
+            None => MAX_PIN_RETRIES,
+            Some((_, entry)) => {
+                debug_assert_eq!(entry.data.len(), 1);
```

Comment:
Does this extra line require moving away from `map_or`?

Reply:
I didn't see a way to include the debug_assert_eq without a match.

Reply:
Doesn't the following work?

```rust
foo.map_or(bar, || {
    debug_assert_eq!(a, b);
    c
});
```

Reply:
That works, if it's clearer. I changed the below use of map_or to the same idiom.